bio_files_processor.py

```diff
@@ -0,0 +1,256 @@
+
+
+def write_seqs_dict(output_file: str, seqs_dict: dict, mode: str = 'w') -> None:
+    """Consequently writes each dictionary value to file as individual line.
+    
+    Args:
+        output_file (str): path to output file.
+        seqs_dict (dict): dictionary containig reads names as keys and tuple
+            with reads array and phred quality array.
+        mode (str): one-letter-string specifiyng write mode, 'w' for 'write'
+            and 'a' for 'append'.
+    
+    Returns:
+        None.
+    """
+    with open(output_file, mode='w') as file_write:
+        for name, seq in seqs_dict.items():
+            file_write.write(f'{name}\n')
+            file_write.write(f'{"".join(seq)}\n')
+
+
+def convert_multiline_fasta_to_oneline(
+        input_file: str,
+        output_filename: str = '',
+        output_path: str = 'results'
+        ) -> None:
+    """Converts multiline fasta sequence to oneline.
+
+    Args:
+        input_file (str): path to input file.
+        output_filename (str): name of output file. Defaults to empty string.
+        output_path (str): name of folder to store output file. Defaults to
+            'results'. 
+
+    Returns:
+        None.
+    """     
+    output_filename = process_paths(input_file, output_filename, output_path)
+    with open(input_file, mode='r') as inp_fa:
+        seqs_dict = {}
+        for line in inp_fa:
+            line = line.strip()         
+            if line.startswith('>'):
+                seq_id = line
+                seqs_dict[seq_id] = []
+            else:
+                seqs_dict[seq_id].append(line)
+            
+    write_seqs_dict(output_filename, seqs_dict)
+
+
+def change_fasta_start_pos(
+        input_file: str,
+        shift: int = 0,
+        output_filename: str = '',
+        output_path: str = 'results'
+        ) -> None:
+    """Shifts starting nucleotide position to desired position for one line fasta file.
+
+    Args:
+        input_file (str): path to input file.
+        shift (int): new position for starting nucleotide. Defaults to zero.
+        output_filename (str): name of output file. Defaults to empty string.
+        output_path (str): name of folder to store output file. Defaults to
+            'results'. 
+
+    Returns:
+        None.
+    """   
+    output_filename = process_paths(input_file, output_filename, output_path)
+    if os.path.isfile(input_file):
+        fa_dict = {}
+        with open(input_file, mode='r') as inp_fa:
+            name = inp_fa.readline().strip()
+            seq = inp_fa.readline().strip()
+            fa_dict[name] = seq[shift:] + seq[:shift]
+            write_seqs_dict(output_filename, fa_dict)
+
+
+def select_genes_from_gbk_to_fasta(
+        input_gbk: str,
+        genes: tuple = (),
+        n_before: int = 3,
+        n_after: int = 1,
+        output_fasta: str = '',
+        output_path: str = ''):
+    """Умеет доставать n трансляций до гена интереса в грязном виде
+    """

```

Comment:
```suggestion
    """
    Умеет доставать n трансляций до гена интереса в грязном виде
    """
```

1) Перенос строки
2) Язык:))
3) В значит "в грязном виде"?